### ISSUE TITLE
tab completion of emoji without arrow keys

### ISSUE THREAD

**Issue by benneti** [vector-im/element-web#19044]
#### What would you like to do?
1. Typing a message
2. trying to type the 😅 emoji via ":swea"
3. hit tab once to select 😅 instead of 😓 and then insert via enter

#### Why would you like to do it?
it is more convenient for me not to use the arrow keys because then I can keep my hands on the home row.

#### How would you like to achieve it?
A setting that toggles whether TAB cycles suggestions or inserts them would be nice.

### Additional context

I think at somepoint TAB to cycle was the default, but I used schildichat for some time so I am not sure whether it was there...

**Comment by dkasak** [comment]
> I think at somepoint TAB to cycle was the default

It was, so isn't this a regression?

**Comment by SimonBrandner** [comment]
@dkasak, This was an intentional change in https://github.com/matrix-org/matrix-react-sdk/pull/5659

**Comment by dkasak** [comment]
Ah, so it is. That's unfortunate. I see you put a thumbs down on the issue, would you be against this being a setting for some reason? It really makes the application quite a bit less usable for some people (including myself).

**Comment by SimonBrandner** [comment]
> Ah, so it is. That's unfortunate.
> 
> I see you put a thumbs down on the issue. Would you be against this being a setting for some reason? It really makes the application quite a bit less usable for some people (including myself).

Somehow didn't realize this wanted it to be a setting, that's alright by me but I don't think it should be the default behavior

**Comment by Peque** [comment]
I find this inconvenient too. However, I still prefer to be able to "confirm" with just TAB.

I think a better fix would be to remember the most frequently used emoji. In my case, when I write `:stuck` I mean :stuck_out_tongue_winking_eye: 99.9 % of the time. I'd just like to avoid using the arrows every time. :joy: 

@benneti Maybe you use :sweat_smile: much more frequently too and this would help? (I do use :sweat_smile: much more frequently, when compared to :sweat:)

**Comment by Peque** [comment]
Cross-referencing https://github.com/vector-im/element-web/issues/18978.

**Comment by t3chguy** [comment]
This is intentional and matches Discord and Slack as well as WAI ARIA guides.
